fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -702,7 +702,7 @@ public class SessionVariable implements Serializable, Writable {
     private boolean enablePipelineEngine = true;
 
     @VariableMgr.VarAttr(name = ENABLE_PIPELINE_X_ENGINE, fuzzy = false, varType = VariableAnnotation.EXPERIMENTAL)
-    private boolean enablePipelineXEngine = false;
+    private boolean enablePipelineXEngine = true;
```

Comment:
not open it by default.
Could add it to fuzzy.
because 2.0 is using old pipeline, we should using master pipeline to test 2.0 and fix bugs.

Reply:
This is only for test. I will turn it off before merging